lib/bindings/AMQPBinding.js

```diff
@@ -111,7 +111,7 @@ function start(callback) {
             }
```

Comment:
An entry in the CHANGES_NEXT_RELEASE file should be added. For instance: 

```
- Fix: avoid connections to AMQP when that transport is not included in configuration (#409)
```

Reply:
> An entry in the CHANGES_NEXT_RELEASE file should be added. For instance:
> 
> ```
> - Fix: avoid connections to AMQP when that transport is not included in configuration (#409)
> ```
Hi @fgalan ,
I have added an entry to CHANGES_NEXT_RELEASE but due to such modification resolve conflict issue is occurring as "This branch has conflicts that must be resolved".

Reply:
I think the conflict is easy to solve.

1. Take the version now in master
2. Add to it the entry: `" Fix: Avoid connections to AMQP and MQTT when these transports are not included in configuration (#409)"` (note the issue number: when a change is related wih an issue number it is a good idea to include it in CHANGES_NEXT_RELEASE also)

Reply:
Github provides a nice GUI to solve small conflicts like that. Do you have the following button at the bottom of the PR? Click "Resolve conflicts"

![imagen](https://user-images.githubusercontent.com/1534240/67105959-15075280-f1ca-11e9-9442-b73801a08564.png)